src/modules/Icon/Icon.tsx

```diff
@@ -32,37 +33,27 @@ const Icon: React.FC<IProps> = ({
   size
 }) => {
   const theme = useTheme();
-  const svgIcon = src || (theme && `assets/icons/${theme.icons[name]}`);
   const [styles, cx] = useStyles(styleSheet);
-  return (
-    <ReactSVG
-      src={svgIcon}
-      className={cx(styles.icon)}
-      beforeInjection={svg => {
-        svg.setAttribute("class", "y-icon");
-        svg.setAttribute("class", `${svg.getAttribute("class")} ${className}`);
-        if (size) {
-          svg.setAttribute("height", size.toString());
-          svg.setAttribute("width", size.toString());
-        }
-        if (height) {
-          svg.setAttribute("height", height.toString());
-        }
-        if (width) {
-          svg.setAttribute("width", width.toString());
-        }
-        if (stroke) {
-          svg.setAttribute("stroke", stroke);
-        }
-        if (fill) {
-          svg.setAttribute("fill", fill);
-        }
-        if (color) {
-          svg.setAttribute("fill", color);
-          svg.setAttribute("stroke", color);
-        }
-      }}
-    />
-  );
+
+  const SvgIcon: any = useMemo(() => {
+    return src || theme.icons[name] || "";
+  }, [src, theme, name]);
+
+  const newProps: any = {};
```

Comment:
A comment explaining why we are building the props this way (to not override svg's values) would be helpful IMO!